src/schema/schema.graphql

```diff
@@ -328,6 +328,14 @@ type Instance {
 	revision: Int # rev
 	status: InviteCompletionStatus
 }
+type Alarm {
```

Comment:
please add mapping in normalize/entities.ts for keys which has shortforms

Reply:
Done